### DISCUSSION TITLE
How to edit a user?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2049] [Help] [answered]
Hello,

I am the admin and I cannot edit myself. I would like to add firstname and lastname.

![image](https://user-images.githubusercontent.com/3023526/126498415-f99a600a-d5e9-4cd3-892f-32b150bc1867.png)

I cannot click in any inputs.

**Comment by GuilhermeHirsch** [comment, marked as answer]
Helo! You can do it this way:

![Sem título](https://user-images.githubusercontent.com/33108596/126498993-3585d2e4-89ea-486e-b2ac-1bda2128214f.jpg)

**Reply by GuilhermeHirsch** [reply]
Through the application, you can create screens that give access to user table information.

![Sem título](https://user-images.githubusercontent.com/33108596/126499876-7021440d-14f8-41b6-a106-787271487dfb.jpg)

**Reply by oswaldB** [reply]
Thank you!

**Reply by GuilhermeHirsch** [reply]
I am here to help, @oswaldB! If it helped, put it as answered, to help other users of the tool in future research.

**Reply by mike12345567** [reply]
Editing your own user is best done through the portal as @GuilhermeHirsch showed, creating/editing user information through an app is something that you can't do right now, ideally you should either edit yourself through the "Update user information" or as an admin update other users information through the Budibase user portal.

Thank you @GuilhermeHirsch for your answer!
